sdk/maps/azure-maps-render/src/test/java/com/azure/maps/render/RenderClientTest.java

```diff
@@ -42,7 +46,10 @@ public void testGetMapTile(HttpClient httpClient, RenderServiceVersion serviceVe
         MapTileOptions mapTileOptions = new MapTileOptions();
         mapTileOptions.setTilesetId(TilesetID.MICROSOFT_BASE_ROAD);
         mapTileOptions.setTileIndex(new TileIndex().setX(10).setY(22).setZ(6));
-        byte[] actualResult = client.getMapTile(mapTileOptions).readAllBytes();
+        InputStream is = client.getMapTile(mapTileOptions);
+        File file = new File("maptilesampleoutput.txt");
```

Comment:
Not allowed.  Tests must not have side effects.

For binary results, simply check that the input stream is not null and its size is not zero. That is enough.